### ISSUE TITLE
Monitor mode not working with TP-Link TL-WN725N

### ISSUE THREAD

**Issue by adarshnin** [mfruba/kernel#44]
I have TP-Link TL-WN725N and i am getting this error ""Error for wireless request "Set Mode" (8B06) : SET failed on device wlan0 ; Invalid argument"".

I'm trying to enable monitor mode...
Does this support monitor mode??

**Comment by edtguy** [comment]
To enable monitor mode on the TP-Link TL-WN725N on a Raspberry Pi running Raspberian, I had to download drivers from this site, http://downloads.fars-robotics.net/

**Comment by vinay2442** [comment]
somebody pls tell how to do it in kali linux

**Comment by sanyam8929** [comment]
pls provide a solution for this

**Comment by vinay2442** [comment]
I'm not able to find one brother..

**Comment by sanyam8929** [comment]
Bro I have just tried to link wn722n V2 drivers for going in to monitor mode ..
You can also try the drivers and let me know if you need any kind of help ..

**Comment by vinay2442** [comment]
But my adapter is 725n, can I put 722 drivers into this hardware, I don't
this it's possible..?

**Comment by sanyam8929** [comment]
Yes I also have tp link wn725n adapter and tp link wn722n boht have same drivers .. the difference is of range .. Yes you can try to put the drivers in Kali Linux and when you are putting the drivers pls make sure you blacklist the old drivers ...

**Comment by vinay2442** [comment]
Can u pls tell me the steps of how to do it?

**Comment by ryangel3** [comment]
Hi guys any update on getting TP-Link TL-WN725N to work with monitor mode on Kali linux?

**Comment by NguyenHau9550** [comment]
> Hi guys any update on getting TP-Link TL-WN725N to work with monitor mode on Kali linux?

im kinda sure that TL-WN725N **cannot** work with monitor mode, maybe you have to buy another device, dont be so sad you're not the only one :))

**Comment by ryangel3** [comment]
I just bought TP-LINK TL-WN722N, I hope this connects. The problem is I have the V3 version and not the v1. Will update the drivers and check if it connects to monitor mode.

**Comment by TECHWITHHACK** [comment]
> I have TP-Link TL-WN725N and i am getting this error ""Error for wireless request "Set Mode" (8B06) : SET failed on device wlan0 ; Invalid argument"".
> 
> I'm trying to enable monitor mode...
> Does this support monitor mode??

Check this video for solution.
https://youtu.be/c4L1sSJzer8

**Comment by TECHWITHHACK** [comment]
> > Hi guys any update on getting TP-Link TL-WN725N to work with monitor mode on Kali linux?
> 
> im kinda sure that TL-WN725N **cannot** work with monitor mode, maybe you have to buy another device, dont be so sad you're not the only one :))

https://youtu.be/c4L1sSJzer8

**Comment by TECHWITHHACK** [comment]
> Hi guys any update on getting TP-Link TL-WN725N to work with monitor mode on Kali linux?
> 

https://youtu.be/c4L1sSJzer8
 Check this video for Solution

**Comment by lucthienphong1120** [comment]
i have the same problem, it doesn't work with kali linux, i tried parrotOS and it worked
![image](https://user-images.githubusercontent.com/90561566/163700411-ad744580-5f74-4a17-9a3d-862571253161.png)

**Comment by TECHWITHHACK** [comment]
kali linux kernel is not supported right now. that's why seem this problem.

**Comment by lucthienphong1120** [comment]
Okay thank you
